lib/mocha.js

```diff
@@ -247,6 +253,28 @@ Mocha.prototype._growl = function(runner, reporter) {
 };
 
 /**
+ * Get require.cache files (node only)
```

Comment:
Hm.. but I thought that `_getRequireCache` will return either cached files (node) or an empty array and that would be okay, won't it? Should I instead check for `process.browser` and don't run this function at all?